Content.Server/Objectives/Conditions/KillPersonCondition.cs

```diff
@@ -0,0 +1,48 @@
+﻿#nullable enable
+using Content.Server.Mobs;
+using Content.Server.Objectives.Interfaces;
+using Content.Shared.GameObjects.Components.Damage;
+using Robust.Shared.Serialization;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Objectives.Conditions
+{
+    public abstract class KillPersonCondition : IObjectiveCondition
+    {
+        protected Mind? Target;
+        public abstract IObjectiveCondition GetAssigned(Mind mind);
+
+        public string Title => $"Kill {Target?.OwnedEntity.Name}";
+
+        public string Description => $"Do it however you like, just make sure they don't last the shift.";
```

Comment:
Also these should probably be loc'ed?
I think you don't even need to use .Name if you loc it, but you could just use {0:theName}?

Reply:
dunno what `{0:theName}` is or how to use it, loc'd it tho